keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -487,10 +489,10 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
 
         test("can retrieve the manifest for the resources") {
           val resource = deliveryConfig.resources.random()
-
           getDeliveryConfig(resource)
             .isSuccess()
-            .isEqualTo(deliveryConfig)
+            .get { name }
+            .isEqualTo(deliveryConfig.name)
```

Comment:
Just because the `DeliveryConfig` object itself won't match due to the addition of `createdAt` in the `metadata` map when reading from the database.